src/components/notice-wrapper/notice-wrapper.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import Snackbar from '@material-ui/core/Snackbar';
+import Slide from '@material-ui/core/Slide';
+
+function TransitionUp(props) {
+  return <Slide {...props} direction='up' />;
+}
+
+const SnackbarMessage = () => {
```

Comment:
Why do we need this component if you use `react-toastify`?

Reply:
I thought i deleted it) . Now it is deleted